### ISSUE TITLE
Warnings after GIT update

### ISSUE THREAD

**Issue by lordganja** [rathena/rathena#760]
Git Hash: '4342221a1e2b28c8731524c77090ba9a0aecf0d8'

`src\map\itemdb.c(617): warning C4700: uninitialized local variable 'ap' used`

```
LINE 614			// Remove from DB`
LINE 615			if (strcmpi(str[1], "clear") == 0 && itemdb_group->remove(itemdb_group, db_ui2key(group_id), &data)) {
LINE 616						va_list ap;
LINE 617				itemdb_group_free(db_ui2key(group_id), &data, ap);
LINE 618				ShowNotice("Item Group '%s' has been cleared.\n", str[0]);
LINE 619				continue;
LINE 620			}
```

EDIT:
Error occurred when I run the server
```
[Error]: skill_parse_row_requiredb: Invalid item (in EQUIP_REQUIRE list)  1549 for skill 2257.
[Error]: sv_readdb: Could not process contents of line 686 of "db/pre-re/skill_require_db.txt".
[Error]: skill_parse_row_requiredb: Invalid item (in EQUIP_REQUIRE list)  2801 for skill 2263.
[Error]: sv_readdb: Could not process contents of line 692 of "db/pre-re/skill_require_db.txt".
```

**Comment by lighta** [comment]
Honnestly I don't see why this is set as confirmed.
Looking at https://raw.githubusercontent.com/rathena/rathena/master/db/pre-re/item_db.txt
The item mentionned : 1549, 2801, are indeed invalide. So if anything that unhide a db_error.

About the va_list, ye I should have initialize it, altough since their implementation differ that annoying ><
Still that just a warning vs a compile error.

**Comment by Lemongrass3110** [comment]
Fixed in 86d6113
